plugins/module_utils/resolver.py

```diff
@@ -221,6 +221,8 @@ def _lookup_ns(self, target):
         return result
 
     def _get_resolver(self, dnsname, nameservers):
+        if self.default_nameservers != self.default_resolver.nameservers:
+            nameservers = self.default_nameservers
```

Comment:
I'm not sure about this change. Why is it necessary?

Reply:
You're correct, this is probably not necessary.